partner_event/tests/test_event_registration.py

```diff
@@ -157,3 +155,65 @@ def test_partner_mobile_and_phone(self):
             self.partner_with_phone_and_mobile.mobile,
             "Incorrect test. Partners phone and mobile must differ",
         )
+
+    def test_update_attendee_partner_id(self):
```

Comment:
Are the new tests needed or they can be merged with existing ones for the missing covered parts?

Reply:
I have removed unnecessary tests, improved others, and added the “_appropriate_” test for `base.partner.merge`.